.vscode/settings.json

```diff
@@ -52,7 +52,8 @@
         "--target-dir=target_ra",
         "--workspace",
         "--message-format=json",
-        "--all-targets"
+        "--all-targets",
+        "--all-features", // --all-features is critical in order to stop crates/re_web_viewer_server/build.rs from building the web viewer
```

Comment:
This is still counter-intuitive to me. I tried reading through `build.rs` and even there it's not obvious why adding "all-features" here would *stop* the web-viewer from being built.

Reply:
Also added: https://github.com/rerun-io/rerun/pull/1624/commits/5948394f6d8d428d01969618e08140d196d33369